extensions/smallrye-openapi-common/deployment/src/main/java/io/quarkus/smallrye/openapi/common/deployment/SmallRyeOpenApiConfig.java

```diff
@@ -27,6 +27,12 @@
     @ConfigItem(defaultValue = "false")
     public boolean alwaysRunFilter;
 
+    /**
+     * Do not include the provided static openapi document (eg. META-INF/openapi.yaml)
+     */
+    @ConfigItem(defaultValue = "false")
+    public boolean ignoreStaticDocument;
```

Comment:
Sorry, I should have been more clear. My main concern was that people might have a static document that they _do_ want to include, but _don't_ want any other static documents to be included from anywhere on the classpath.
Does that make sense?

I'm no saying that this property doesn't make sense, I just think it might need more granularity.
Maybe something like a `StaticDocumentInclusion` enum with:
- OFF
- CURRENT_MODULE (?)
- ALL

Reply:
That can work, let me change that.

Reply:
I would go with the current solution for now, and extend later with static document selection / merge.

Reply:
@phillip-kruger After talking to @mickroll I think ALL doesn't make sense and I think my main concern is already covered because you are using `getResourceAsStream()` and not something like `getResources()` to get everything from the CP.
This will pick up files from META-INF of the current module/jar and will only descend deeper into the CP if nothing is found. 👍 

Sorry for the noise!

Reply:
Yes CP should pickup one from module first. So I wonder if we should not leave this as is ?

Reply:
Yeah, that's what I tried to say on Zulip 😉

Reply:
OK. Cool. Let's leave as is.